compiler/rustc_feature/src/accepted.rs

```diff
@@ -159,6 +159,9 @@ declare_features! (
     (accepted, explicit_generic_args_with_impl_trait, "1.63.0", Some(83701), None),
     /// Allows arbitrary expressions in key-value attributes at parse time.
     (accepted, extended_key_value_attributes, "1.54.0", Some(78835), None),
+    /// Allows using `efiapi`, `aapcs`, `sysv64` and `win64` as calling
+    /// convention for functions with varargs.
+    (accepted, extended_varargs_abi_support, "1.74.0", Some(100189), None),
```

Comment:
As noted in https://github.com/rust-lang/rust/pull/116161#discussion_r1336437170
```suggestion
    (accepted, extended_varargs_abi_support, "CURRENT_RUSTC_VERSION", Some(100189), None),
```